### ISSUE TITLE
[Request] Indicator for results

### ISSUE THREAD

**Issue by AR15senalOfCode** [valarnin/NTRRemoteDebugger#18]
It would be cool if there was an indicator which shows how much results the program found yet (during the 'next scan'-process).

**Comment by f-israel** [comment]
If I'm not wrong you can take the label above the final result as indicator. As addresses are filtered out the num is decreasing. This should give a hint about the progress.

**Comment by valarnin** [comment]
Correct. The number of results found is displayed as a label and is updated in real time as the scan is performed. Unless you're saying that this functionality is no longer working as intended?

**Comment by AR15senalOfCode** [comment]
@valarnin I don't know if that is the case. I created this thread one year ago but I am currently unable to test this out. It may not have been implemented at that time or I was just blind but yeah this thread is one year old, just sayin...
